macro/run_simScript.py

```diff
@@ -165,7 +161,9 @@
 if (HNL and options.RPVSUSY) or (HNL and options.DarkPhoton) or (options.DarkPhoton and options.RPVSUSY): 
  print("cannot have HNL and SUSY or DP at the same time, abort")
  sys.exit(2)
-
+if inclusive=='meson' and options.MM not in ['pi0','eta','omega','eta1','eta11']:
+ print("Please choose a DP sources from one of these (pi0, eta, omega, eta1, eta11) for the Meson Production")
+ sys.exit()
```

Comment:
You can use [argparse choices](https://docs.python.org/3/library/argparse.html#choices) to enforce this automatically, so that you don't need to do it manually.